packages/core/src/error-reporter.ts

```diff
@@ -9,6 +9,14 @@ import { createHash } from 'node:crypto';
 import type { InstanceType } from './InstanceSettings';
 import { Logger } from './logging/logger';
 
+type ErrorReporterInitOptions = {
+	serverType: InstanceType | 'task_runner';
+	dsn: string;
+	release?: string;
+	environment?: string;
+	serverName?: string;
```

Comment:
I don't these three being passed in anywhere.
Do we plan to add these on existing `init` calls? 
if yes, can we do that in the same PR? 
if not, I think we should revert the changes below to continue picking these only from `process.env`.

Reply:
They're being passed in by the runner [here](https://github.com/n8n-io/n8n/pull/12495/files#diff-58e8ed61d5707ef626e062bdee9e442e06339a566d6d9a6013955c7829f5f419R62), that's the fix!

Reply:
ah, in that case, do we still need the `?? process.env.` bit ?

Reply:
So the runner is passing its own config values, but n8n (cli) has always read these three `release`, `environment`, `serverName` from its environment, so `?? process.env.*` keeps the behavior in cli as it was.

Reply:
but won't the values in `SentryConfig` be identical to the ones in `process.env` ?
perhaps I'm reading this wrong, but to me the `?? process.env.*` bit looks like dead code.

Reply:
Yeah let's make it so that there's only one way to define them: either 1) passing as opts or 2) taken directly from process.env. That way there isn't two separate ways to manage these